src/notebook/index.css

```diff
@@ -37,67 +37,87 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 5.6px;
```

Comment:
Though I do remember @ellisonbg talking about making everything a multiple of 4 at some point.